recipes-core/acrn/files/0001-acrn-config-append-kconfig-setting-on-new-board.patch

```diff
@@ -0,0 +1,31 @@
+From 0eccde3eb907a74c46a58ff827bbfc2bb90dadce Mon Sep 17 00:00:00 2001
+From: Victor Sun <victor.sun@intel.com>
+Date: Wed, 5 Feb 2020 16:43:00 +0800
+Subject: [PATCH] acrn-config: append kconfig setting on new board
+
+When user has already prepared some default kconfig settings in
+hypervisor/arch/x86/configs/$(BOARD).config file even the board is new
+to Acrn source, the acrn-config will keep these settings and append other
+suggested settings on the end of the $(BOARD).config file.
+
+Signed-off-by: Victor Sun <victor.sun@intel.com>
```

Comment:
No Upstream-Status.

Reply:
yes, this is short time solution for v1.5.1. 
might not needed for future version.